manage_rbac.py

```diff
@@ -267,17 +267,22 @@ def k8s_client(self):
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Updates RBAC cluster role bindings and role bindings.')
     parser.add_argument('--config', help='YAML configuration file to load')
+    parser.add_argument('--namespace', help='Namespace for service accounts', default=os.environ.get('NAMESPACE'))
     parser.add_argument('--kubectl-auth', action='store_true', help='Use kubectl command to refresh auth (useful for GKE)')
     args = parser.parse_args()
     if args.kubectl_auth:
         os.system('kubectl get ns >/dev/null 2>&1')
     try:
         if args.config is not None:
             logging.debug("Updating RBAC from file.")
-            RBACManager().update(file=args.config)
+            RBACManager(args.namespace).update(file=args.config)
         else:
+            if not args.namespace:
+                logging.error("A specified namespace is required when running in controller-mode")
+                exit(parser.print_usage())
+            logging.info("Managing service accounts in namespace " + args.namespace)
             logging.debug("Starting controller.")
-            RBACManager().controller()
+            RBACManager(args.namespace).controller()
```

Comment:
Why namespace the controller? Do you not want a single controller to be able to apply RBACdefinitions in other namespaces?

Reply:
I don't think it was doing that anyway. The watch was limited to the `rbac-manager` namespace. More philosophically, there's a bigger discussion to be had...starting that outside comments...

Reply:
Understood

Reply:
Sync complete. Eventually we'll move toward a cluster CRD instead of namespaced, but for now, making sure all the rbacdefs are in the same namespace as the pod is the way to go.

Reply:
Moving towards a cluster CRD is not a given.